06-Branch/readme.md

```diff
@@ -39,14 +39,14 @@ So, to sum up what we've just learned
 
 * git branch [--list]
 	* list branches
-* git branch $BRANCH_NAME
-	* creates branch named $BRANCH_NAME from current HEAD
-* git branch -m $OLD_BRANCH_NAME $NEW_BRANCH_NAME
+* git branch _BRANCH_NAME_
+	* create branch named _BRANCH_NAME_ from current HEAD
+* git branch -m _OLD_BRANCH_NAME_ _NEW_BRANCH_NAME_
 	* rename branch
-* git branch -d $BRANCH_NAME
+* git branch -d _BRANCH_NAME_
 	* delete branch
 
-More informations
+More information
```

Comment:
Remember to remove one of the dashes below this too!  (I also fixed this in PR #28)